apps/hub/src/ai/data/storage.ts

```diff
@@ -31,10 +31,18 @@ export function decodeDbWorkflowToClientWorkflow(
     return {
       id: row.id,
       timestamp: row.createdAt.getTime(),
-      status: "error",
+      status: "finished",
       inputs: {},
       steps: [],
-      result: `Invalid workflow format in the database: ${e}`,
+      result: {
+        code: "",
+        isValid: false,
+        totalPrice: 0,
+        runTimeMs: 0,
+        llmRunCount: 0,
+        logSummary: "",
```

Comment:
Explicitly writing these is annoying, can be improved in the future with more clever types but I didn't want to spend too much time on this.